src/lib/autoinstall/widgets/storage/disk_device.rb

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm/common_widgets"
+
+module Y2Autoinstallation
+  module Widgets
+    module Storage
+      # Widget to select a block device for a partition section
+      #
+      # It corresponds to the `device` element in the profile.
+      class DiskDevice < CWM::ComboBox
+        # Constructor
+        def initialize
+          textdomain "autoinst"
+          super
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          _("Device")
+        end
+
+        # @macro seeAbstractWidget
+        def opt
+          [:editable]
+        end
+
+        DISKS = [
+          "/dev/sda", "/dev/sdb", "/dev/vda", "/dev/vdb", "/dev/hda", "/dev/hdb"
```

Comment:
Do you think this still makes sense? We don't make suggestions for the format of other device names. And to be honest, this list of values looks like taken from year 2001.

Reply:
To be honest I overlooked this because it was already done. But, answering your question, no, I don't think.

So, I'll remove it to have more consistence among options.